CHANGES/5488.misc

```diff
@@ -0,0 +1 @@
+Docs update to use `pulp_use_system_wide_pkgs`.
```

Comment:
@pavelpicka , maybe it makes sense to do it a .doc, see all options  for towncrier-based release notes https://github.com/hawkowl/towncrier#news-fragments.

Reply:
it is a doc update so .doc should be there